src/toil_scripts/gatk_germline/germline_config.py

```diff
@@ -0,0 +1,103 @@
+import textwrap
+
+
+def generate_config():
+    return textwrap.dedent("""
+        # GATK Germline Pipeline configuration file
+        # This configuration file is formatted in YAML. Simply write the value (at least one space) after the colon.
+        # Edit the values in this configuration file and then rerun the pipeline
+        # Comments (beginning with #) do not need to be removed. Optional parameters may be left blank.
+        ##############################################################################################################
+        # Required: Reference Genome URL (hg19.fa)
+        genome-fasta:
+
+        # Optional: Reference Genome Index URL (hg19.fa.fai)
+        genome-fai:
+
+        # Optional: Reference Genome Sequence Dictionary (hg19.dict)
+        genome-dict:
+
+        # Optional: URL (1000G_phase1.indels.hg19.sites.fixed.vcf)
+        phase:
+
+        # Optional: URL (Mills_and_1000G_gold_standard.indels.hg19.sites.vcf)
+        mills:
+
+        # Optional: URL (dbsnp_138.hg19.excluding_sites_after_129.vcf)
+        dbsnp:
+
+        # Optional: URL (hapmap_3.3.hg19.sites.vcf)
+        hapmap:
+
+        # Optional: URL (1000G_omni2.5.hg19.sites.vcf)
+        omni:
+
+        # Optional: (boolean) Run BWA on fastqs
+        run-bwa:
+
+        # Optional. If true, BWA trims adapters
+        trim: True
+
+        # Optional: Reference fasta file (amb) -- if not present will be generated
+        amb:
+
+        # Optional: Reference fasta file (ann) -- If not present will be generated
+        ann:
+
+        # Optional: Reference fasta file (bwt) -- If not present will be generated
+        bwt:
+
+        # Optional: Reference fasta file (pac) -- If not present will be generated
+        pac:
+
+        # Optional: Reference fasta file (sa) -- If not present will be generated
+        sa:
+
+        # Optional: Alternate file for reference build (alt). Necessary for alt aware alignment
+        alt:
+
+        # Optional: (boolean) Run GATK Preprocessing
+        preprocess:
+
+        # Optional: (boolean) Run GATK VQSR
+        run-vqsr:
+
+        # Optional: (boolean) Joint Calling
+        joint:
+
+        # Optional: Approximate input file size
```

Comment:
Agreed. It's impossible to have a reasonable default for this.

However, why does this need to be present at all? Can't we examine the input file for its size?